mitmproxy/tools/console/master.py

```diff
@@ -120,12 +121,33 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        editors = [
+            "ghex",
+            "bless",
+            "xxd",
+            "imhex",
+            "hexedit",
+            "hexed",
+            "hxd",
+            "hexdump",
+            "emacs",
+        ]
+        for editor in editors:
+            if shutil.which(editor):
+                return editor
+        return "vi"
```

Comment:
Can we return `self.get_editor()` instead? We don't want people to get stuck in vi. :)

Reply:
Don't know how I missed that! I'll update the PR